### PR TITLE
gvr-avatar with bvh assets

### PR DESCRIPTION
**Pull request by spodila1**
gvr-avatar: Added the following assets in YBot folder.

Model -- YBOT.fbx
BvhAnimations -- Samba_Dancing_mixamo.com.bvh, Zombie_Stand_Up_mixamo.com, Football_Hike_mixamo.com

GearVRf-DCO-1.0-Signed-off-by: Sahithi Podila sah.pod@partner.samsung.com

### CODE REVIEW

File: gvr-avatar/app/src/main/java/org/gearvrf/avatardemo/AvatarMain.java [hunk 1]
```diff
@@ -27,45 +28,45 @@
...
-//    private final String mModelPath = "DeepMotion/sahithi_character_skin.fbx";
-//    private final String mAnimationPath = "bodyturn.txt";
-//    private final String mModelPath = "Andromeda/Andromeda.dae";
-//    private final String[] mAnimationPaths =  { "Andromeda/HipHopDancing.dae", "Andromeda/Bellydancing.dae", "Andromeda/Boxing.dae" };
-//    private final String[] mAnimationPaths = { "DeepMotion/animation_baked.fbx" };
-//    private final String mModelPath = "Lily/female_outfitJ.fbx";
-//    private final String[] mAnimationPaths = { "Lily/Idle.fbx" };
-    private final String mModelPath = "astroboy/astro_boy.dae";
-    private final String[] mAnimationPaths =  { "astroboy/astro_boy.dae" };
+public class AvatarMain extends GVRMain {
```
Comment (liaxim) [inline gvr-avatar/app/src/main/java/org/gearvrf/avatardemo/AvatarMain.java]: See gvr-x3d-demo and how we support setting the model to load via a system property (org.gearvrf.gvr-x3d-demo). Then all that commented out code can be removed and the models changed quicker.

**Comment by liaxim** [comment]
Doesn't work for me. Produces this at startup:


If gvr.xml is really not needed remove the reference to it from AvatarActivity.

**Comment by spodila1** [comment]
Please tap on the screen to see the model with animation.

**Comment by NolaDonato** [comment]
The FPS is low because the BVH importer prints out a lot of stuff. I removed this.

**Comment by NolaDonato** [comment]
If you download the YBot from mixamo without animation and use that FBX in the demo, the YBot will show up immediately when it loads.

**Comment by NolaDonato** [comment]
I figured out why it crashes too. If you tap before the avatar is finished loading it will crash. I will fix this.

**Comment by liaxim** [comment]
Doesn't build for me:
```
:gvr-avatar:javaPreCompileDebug
/home/mihail/gvrfmaster/GearVRf-Demos/gvr-avatar/app/src/main/java/org/gearvrf/avatardemo/AvatarMain.java:64: error: <anonymous org.gearvrf.avatardemo.AvatarMain$1> is not abstract and does not override abstract method onModelLoaded(GVRSceneObject,String,String) in IAvatarEvents
    private GVRAvatar.IAvatarEvents mAvatarListener = new GVRAvatar.IAvatarEvents() {
                                                                                    ^
/home/mihail/gvrfmaster/GearVRf-Demos/gvr-avatar/app/src/main/java/org/gearvrf/avatardemo/AvatarMain.java:124: error: method loadAnimation in class GVRAvatar cannot be applied to given types;
        mAvatar.loadAnimation(new GVRAndroidResource(mContext, animPath));
               ^
  required: GVRAndroidResource,String
  found: GVRAndroidResource
  reason: actual and formal argument lists differ in length
2 errors
:gvr-avatar:compileDebugJavaWithJavac FAILED
```

**Comment by NolaDonato** [comment]
I have worked a bit on this demo to address comments. Now the demo loads animations and loops automatically. It runs at 60fps for me on the S8 with the YBot. I remove Andromeda and the large textures. The demo is currently set up to retarget animation from a BVH with a different skeleton to the YBot.

**Comment by liaxim** [comment]
gvr-physicsload doesn't build.

**Comment by thomasflynn** [comment]
gvr-physicsload builds now.

**Pull request merged**